source/guides/network/supernode-enrollment.rst

```diff
@@ -0,0 +1,17 @@
+##################################
+Enrolling to the Supernode program
+##################################
+
+**********************
+Using symbol-bootstrap
+**********************
+
+********
+Manually
+********
+
+- Enrols as a supernode (msg on chain), content ``enrol ${nodePublicKey} ${agentUrl}``
+
+  - supernodeEnrolmentAddress: TDETDMT5S2ADAYGJXPATUHQUYVGSLSVJ6TLSXQQ for TESTNET
+
+- Installs an agent to monitor it. source https://github.com/nemgrouplimited/symbol-node-rewards, binary in S3, url??? platforms???
```

Comment:
The agent binary can be publically downloaded from 

https://symbol-node-reward.s3-eu-west-1.amazonaws.com/agent_binary/agent-linux.bin

Atm, we are just deploying the Linux version of the binary, we would need to upgrade adding other distributions. Unsure if they are going to be windows or mac supernodes. We would need to test the agents on the different distributions. 

Note, bootstrap pulls (and caches) this Linux binary from the s3 location when setting up the node. When using --upgrade, the latest binary will be pulled as part of the node upgrade.